diff --git a/node_modules/ecpair/src/cjs/networks.cjs b/node_modules/ecpair/src/cjs/networks.cjs
index 1a92a8f..1a2a95c 100644
--- a/node_modules/ecpair/src/cjs/networks.cjs
+++ b/node_modules/ecpair/src/cjs/networks.cjs
@@ -2,15 +2,15 @@
 Object.defineProperty(exports, '__esModule', { value: true });
 exports.testnet = exports.bitcoin = void 0;
 exports.bitcoin = {
-  messagePrefix: '\x18Bitcoin Signed Message:\n',
-  bech32: 'bc',
+  messagePrefix: '\x14XEP Signed Message:\n',
+  bech32: 'ep',
   bip32: {
     public: 0x0488b21e,
     private: 0x0488ade4,
   },
-  pubKeyHash: 0x00,
-  scriptHash: 0x05,
-  wif: 0x80,
+  pubKeyHash: 0x37,
+  scriptHash: 0x89,
+  wif: 0xa2,
 };
 exports.testnet = {
   messagePrefix: '\x18Bitcoin Signed Message:\n',
diff --git a/node_modules/ecpair/src/esm/networks.js b/node_modules/ecpair/src/esm/networks.js
index bc3cccb..a616802 100644
--- a/node_modules/ecpair/src/esm/networks.js
+++ b/node_modules/ecpair/src/esm/networks.js
@@ -1,13 +1,13 @@
 export const bitcoin = {
-  messagePrefix: '\x18Bitcoin Signed Message:\n',
-  bech32: 'bc',
+  messagePrefix: '\x14XEP Signed Message:\n',
+  bech32: 'ep',
   bip32: {
     public: 0x0488b21e,
     private: 0x0488ade4,
   },
-  pubKeyHash: 0x00,
-  scriptHash: 0x05,
-  wif: 0x80,
+  pubKeyHash: 0x37,
+  scriptHash: 0x89,
+  wif: 0xa2,
 };
 export const testnet = {
   messagePrefix: '\x18Bitcoin Signed Message:\n',
